transitions/loop.h

```diff
@@ -103,4 +103,69 @@ class TrLoopUntil {
   }
 };
 
+// Usage: TransitionLoopWhileL<LOOP_TRANSITION, END_TRANSITION, CONDITION>
+// LOOP_TRANSITION : TRANSITION
+// END_TRANSITION : TRANSITION
+// return value: COLOR
+//
+// Continuously runs LOOP_TRANSITION while CONDITION > 0
+// Makes more sense if TRANSITION is a TrConcat, as this will
+// transition to/from the intermediate steps in a loop.
+// If CONDITION <= 0, runs END_TRANSITION and stops.
+
+template<class LoopTr, class EndTr, class CONDITION>
+class TransitionLoopWhileL {
+
+public:
+void run(BladeBase* blade) {
```

Comment:
this whole function should be indented two steps

Reply:
fixed